index.bs

```diff
@@ -356,17 +368,17 @@ A <dfn>registry entry</dfn> is a record with the following fields:
   1. ReturnIfAbrupt(_status_).
   1. Assert: _entry_'s whole dependency graph is in "ready" state.
   1. Return _entry_.
-</pre>
+</emu-alg>
 
 <h4 id="request-ready">RequestReady(loader, key)</h4>
 
-<pre emu-alg>
+<emu-alg aoid="RequestReady">
```

Comment:
yeah, I noticed that. It is definitely nicer since the link will jump to the header instead of the first line of the algo. I can do that.